docs/cstor-volume-expansion.md

```diff
@@ -0,0 +1,228 @@
+old_desc_blocks = 3, new_desc_blocks = 4
+The filesystem on /dev/sdc is now 7864320 (4k) blocks long.
+```
+
+## Verify size in application pod
+
+Verify if the size of the volume is updated inside application pod using the following command:
+
+```
+kubectl exec -it <app_pod_name> -n <application-namespace> -- df -h
+```
+Example command:
+```
+kubectl exec -it busybox-555775f56c-rf4qt -n default -- df -h
+```
+Example output:
+```
+Filesystem                Size      Used Available Use% Mounted on
+overlay                  28.9G      3.1G     25.8G  11% /
+tmpfs                    64.0M         0     64.0M   0% /dev
+tmpfs                     3.6G         0      3.6G   0% /sys/fs/cgroup
+/dev/sda1                28.9G      3.1G     25.8G  11% /dev/termination-log
+/dev/sda1                28.9G      3.1G     25.8G  11% /etc/resolv.conf
+/dev/sda1                28.9G      3.1G     25.8G  11% /etc/hostname
+/dev/sda1                28.9G      3.1G     25.8G  11% /etc/hosts
+shm                      64.0M         0     64.0M   0% /dev/shm
+/dev/sdc                 29.4G     44.0M     29.3G   0% /var/lib/mysql
+tmpfs                     3.6G     12.0K      3.6G   0% /var/run/secrets/kubernetes.io/serviceaccount
+tmpfs                     3.6G         0      3.6G   0% /proc/acpi
+tmpfs                    64.0M         0     64.0M   0% /proc/kcore
+tmpfs                    64.0M         0     64.0M   0% /proc/keys
+tmpfs                    64.0M         0     64.0M   0% /proc/timer_list
+tmpfs                    64.0M         0     64.0M   0% /proc/sched_debug
+tmpfs                     3.6G         0      3.6G   0% /proc/scsi
+tmpfs                     3.6G         0      3.6G   0% /sys/firmware
+```
+From the above output, capacity of corresponding device where application is mounted has been updated as `30G`.
+
+## Edit PV
+
+The capacity of corresponding PV has to be changed manually. This can be done by editing the PV and update the value in `spec.capacity.storage` field.
+
+```
+kubectl get pv
+```
+Example output:
+```
+NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM                     STORAGECLASS     REASON   AGE
+pvc-0546bc97-5e64-4f94-b18e-6929f615fb75   20Gi       RWO            Delete           Bound    default/demo-vol1-claim   openebs-sc-new            9m26s
+```
+### Patch capacity of the PV:
+```
+kubectl edit pv <pv_name>
+```
+Example command:
+```
+kubectl edit pv pvc-0546bc97-5e64-4f94-b18e-6929f615fb75
+```
+Now, verify the update PV capacity details using the following command:
+```
+kubectl get pv
+```
+Example output:
+```
+NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM                     STORAGECLASS     REASON   AGE
+pvc-0546bc97-5e64-4f94-b18e-6929f615fb75   30Gi       RWO            Delete           Bound    default/demo-vol1-claim   openebs-sc-new            10m
+```
```

Comment:
Add following 
**Note:** PVC capacity will be inconsistent with PV capacity. This will be fixed during migration of Non-CSI volumes to CSI volumes.

Reply:
@mittachaitu Added. Also I added `Non-CSI` Infront of cStor volume in the Introduction section. Please check if that is also fine....